packages/graBing/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname="graBing"
+pkgver=9.bb8b08b
+pkgrel=1
+pkgdesc="Counts all the hostnames for an IP adress"
+arch=('x86_64' 'i686')
+url="https://github.com/black-brain/graBing"
+license=('GPL3')
+groups=('blackarch-fingerprint' 'blackarch-scanner')
```

Comment:
from:
`groups=('blackarch-fingerprint' 'blackarch-scanner')`
to:
`+groups=('blackarch' 'blackarch-fingerprint' 'blackarch-scanner')`

Reply:
Ok, I will fix it .